contracts/lib/LibLiquidation.sol

```diff
@@ -138,4 +161,32 @@ library LibLiquidation {
             return amountToReturn;
         }
     }
+
+    /**
+     * @return true if the margin is greater than 10x liquidation gas cost (in quote tokens)
+     * @dev Assumes params are WAD except liquidationGasCost
+     * @param liquidatedBaseChange How much base token to be liquidated
+     * @param liquidatedQuoteChange How much quote token to be liquidated
+     * @param balanceToBeLiquidated The balance of account to be liquidated
+     * @param liquidationGasCost Approximately how much gas is used to call liquidate()
+     * @param price Current fair price
+     */
+    function partialLiquidationIsValid(
+        int256 liquidatedBaseChange,
+        int256 liquidatedQuoteChange,
+        Balances.Account memory balanceToBeLiquidated,
+        uint256 liquidationGasCost,
+        uint256 price
+    ) internal pure returns (bool) {
+        uint256 minimumLeftoverMargin = PRBMathUD60x18.mul(
+            balanceToBeLiquidated.lastUpdatedGasPrice,
+            liquidationGasCost
+        ) * 10;
```

Comment:
Whats this magic number 10 here?

Reply:
Yeah true this is a magic number. Do you think this should be a market parameter? This is just what Garner and webby agreed on.

Reply:
So minimum leftover margin must be greater than 10 * liquidation gas cost, and min margin is some value + 6 * liquidation gas cost so this may or may not leave them above min margin?

Yeah I think set this as a constant for now. We could even make it configurable but lets keep it simple for now as a const.

Reply:
Even if it's hardcoded, it needs to be a defined constant somewhere.